site/content/docs/4.3/about/brand.md

```diff
@@ -31,25 +31,25 @@ Use either the Bootstrap mark (a capital **B**) or the standard logo (just **Boo
 
 ## Download mark
 
-Download the Bootstrap mark in one of three styles, each available as an SVG file. Right click, Save as.
+Download the Bootstrap mark in one of three styles, each available as an SVG file. Click to download the logos.
```

Comment:
Or make the new phrase bold maybe. Not sure it's good, BTW. Maybe it should say "Click on each logo to download it." or something. Let's wait for someone else's feedback.